kin-sdk-core/src/main/java/kin/sdk/core/EncryptedAccount.java

```diff
@@ -0,0 +1,21 @@
+package kin.sdk.core;
+
+
+class EncryptedAccount {
```

Comment:
maybe change to just: Account?

Reply:
Naming is hard... Maybe `StellarAccount`?

Reply:
Simple as possible that's what I think, `Account` like we have on Ethereum

Reply:
the problem is we have KinAccount, I want to make a distinction between this objects, geth has their own `Account` on their API

Reply:
I still think it should be `Account` :)
👍

Reply:
ok, I'll make it simple